### DISCUSSION TITLE
tanstack react query 사용

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#68] [General]
#54

**Comment by gayeong0916** [comment]
오늘 tanstack & Context API 와 tanstack & zustand 중에 어떤 걸 쓰지 회의하나요?!?!

**Comment by JiinSeok** [comment]
- tanstack
- zustand는 프로젝트 전역
- Context는 UI 컴포넌트가 받은 props를 안에서 공유하는 용도

**Reply by JiinSeok** [reply]
프로젝트 글로벌이 아닌 컴포넌트가 사용하는 상태에 대해서는 더 논의 @gayeong0916

**Reply by JiinSeok** [reply]
현재는 scss 변수를 이용할 수 있는 반응형 상태 외에는 컴포넌트 내부 전역 상태가 필요하지 않아서 context api 사용 중단
